accounts/get-exams.js

```diff
@@ -0,0 +1,80 @@
+/* eslint-disable no-process-exit */
+
+// After running the script and getting the JSON output,
+// you need to manually remove the last comma in the output file
+
+require("dotenv").config();
+
+const Stream = require("stream");
+const fs = require("fs");
+const mongodb = require("mongodb");
+const ora = require("ora");
+
+const MongoClient = mongodb.MongoClient;
+
+const outputFile = fs.createWriteStream("exams.json", { encoding: "utf8" });
+
+outputFile.write("[\n");
+
+const rs = new Stream.Readable({ objectMode: true });
+rs._read = function () {};
```

Comment:
This doesn't seem necessary. Since mongo gives you a readable stream, can't you just use that?

Reply:
Yea, I'd rather just leave it for now if that works. I'm not sure how much we will use this script. Just thought it would be nice to have it on file in case we need it again - if we use it again, I can take a look at refactoring that.

Reply:
Fair enough!